account_bank_statement_import_adyen/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+======================
+Adyen statement import
+======================
+
+This module processes Adyen transaction statements in xlsx format. You can
```

Comment:
Please document what file exactly is handled by this module. I am trying to convert this to 13.0, so to test downloaded the payment accounting report (after login to Ayden: Finance ==> Downloadable reports ==> Payment Accounting). That has 34 columns. Your test file has 31 columns, as is also tested in the code. I cannot find a report that online that looks like the one the code is expecting.

Reply:
OK I found it. It is the settlement details report from the merchant level. Please document this in the readme.